src/main/java/donk/Ui.java

```diff
@@ -0,0 +1,106 @@
+package donk;
+
+import donk.task.Task;
+import donk.task.TaskList;
+import donk.task.TaskType;
+
+public class Ui {
+
+    public Ui() {
+
+    }
+
+    /**
+     * Displays an error message indicating that no save file was found
+     * and a new task list will be started.
+     */
+    public void showLoadingError() {
+        System.out.println("No save file found, starting new task list");
+    }
+
+    /**
+     * Greets the user with a welcome message when the application starts.
+     */
+    public void greet() {
+        String greetMsg = " ____________________________________________________________\n" +
+                " Hello! I'm donk.Donk, the super intelligent chatbot\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n";
+
+        System.out.println(greetMsg);
+    }
```

Comment:
I noticed that all `Ui.java` methods do not require instantiated variables inside the class, so technically Ui does not even have to be instantiated as an object.  Do you think it would be better make `Ui.java` methods static instead? 

An implication of this would be that Ui no longer has to be stored as an object inside `Donk.java`. Instead, these methods can just be called by doing `Ui.parse(...)`, `Ui.bye()`, etc.

Reply:
That's a good point!